src/processor/map/map_copy_to.cpp

```diff
@@ -29,7 +29,8 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapCopyTo(LogicalOperator* logical
     auto sharedState = exportFunc.initShared(*clientContext, *bindData);
     auto info =
         CopyToInfo{exportFunc, std::move(bindData), std::move(vectorsToCopyPos), std::move(isFlat)};
-    auto printInfo = std::make_unique<OPPrintInfo>(logicalCopyTo.getExpressionsForPrinting());
+    auto printInfo =
+        std::make_unique<CopyToPrintInfo>(info.bindData->names, info.bindData->fileName);
```

Comment:
names is a very vague naming. Ask @acquamarin to see if it is column names or not. If so, rename to column names